coldfront/core/portal/views.py

```diff
@@ -35,20 +34,26 @@ def home(request):
 
         cluster_access_attributes = AllocationUserAttribute.objects.filter(allocation_attribute_type__name='Cluster Account Status',
                                                                        allocation_user__user=request.user)
-
         access_states = {}
         for attribute in cluster_access_attributes:
             project = attribute.allocation.project
             status = attribute.value
             access_states[project] = status
 
         abc_projects, savio_projects, vector_projects = set(), set(), set()
+
         for project in project_list:
             project.display_status = access_states.get(project, None)
 
             if project.display_status is not None and 'Active' in project.display_status:
                 context['cluster_username'] = request.user.username
 
+            # crashed on login if part of a project and the name was None
```

Comment:
I'm not sure this would happen. (Pretty sure this block is left over from testing?)

Reply:
@jofeinstein

Reply:
the line that caused the crash were below the code I put in: Line 59.

if project.name == 'abc':
    abc_projects.add(project.name)
elif project.name.startswith('vector_'):
    vector_projects.add(project.name)
else:
    savio_projects.add(project.name)

If project.name == None, there was an error bc None does have the startswith method. 

I don't know if you want to remove this chunk of code.

Reply:
@matthew-li

Reply:
You're using the test data provided with ColdFront; none of the `Projects` there have anything in the `name` field because that field was added by us after the fact.

In production, every `Project` will have a `name`. I'd recommend running the following in the shell:

```
from coldfront.core.project.models import Project

for project in Project.objects.all():
    project.name = project.title
    project.save()
```

(I'm hoping to come up with an easy way for us to load the production database into our VMs, so that we can work with real data.)

Reply:
Got it. Ill remove it and push the code